apps/src/templates/YourSchool.jsx

```diff
@@ -4,6 +4,16 @@ export default class YourSchool extends React.Component {
   render() {
     return (
       <div>
+        <h1>
+          Bring Computer Science to your school district
+        </h1>
+        <h3>
+          Our vision is that every student in every school should have the opportunity to learn computer science.
+          Help us by letting us know how computer science is taught at your school and by pledging to do more.
+        </h3>
+        <h2>
+          Tell us about your school
+        </h2>
```

Comment:
You'll also presumably eventually want these localized, though it's fine to do that in a future PR.

Reply:
Yeah, I plan to do that when I do more work on this component. Thanks for the reminder!